src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -1329,6 +1329,9 @@ public Value tryParseLocalDate(DateTimeFormatter formatter) {
      * @return the parsed LocalDate wrapped as a Value, or the Value itself if it was empty or a parsing error occurred
      */
     public Value tryParseLocalDate(String languageCode) {
+        if (Strings.isEmpty(languageCode)) {
```

Comment:
in every other location, we'd use getCurrentLang() here ?! is this consistent?

Reply:
This method only **tries** to parse the value as a LocalDate. So when an empty language is passed, there is no need to try any further.

I think for a `tryParseLocalDate` returning without defaulting to the current language is the expected behavior. If this was a `parseLocalDate`, then the default should be used.

Reply:
Still in all other methods "null" is used to represent the default language. And the try is meant to "tryParse" not to check parameters(?). I'd say if parseLocalDate yields a value, tryParseLocalDate with **the same parameters** should also return a value.